cluster/ci/monitoring/Makefile

```diff
@@ -3,7 +3,7 @@ prometheus-k8s_token := $(shell oc get --as=system:serviceaccount:ci:config-upda
 .PHONY : prow-monitoring-deploy
 prow-monitoring-deploy: ci-secret-deploy
 	@echo "Deploying prow-monitoring stack ..."
-	oc apply -f ./prometheus_expose.yaml
+	oc process -n prow-monitoring -f ./prometheus_expose.yaml | oc apply -f -
```

Comment:
Does the namespace arg matter for `oc process` and don't we need it for `oc apply` as well? Why isn't this part of `core-services` and can we move  there instead of changing this?

Reply:
* namespace as param (wont work otherwise): https://docs.openshift.com/container-platform/3.11/dev_guide/templates.html#writing-object-list
* specify `-n` in process command: process need a BRAC to run: the namespace determines in which namespace we want to run it and hence in which namespace we need to add the RBAC
* Yes, we could move it into core-services since it uses template. I want to move all the file in one shot when we are not under the pressure of migrating to build farm.